### PR TITLE
my edits to Pierre's aws-container document

### PR DESCRIPTION
**Pull request by ghost**
editing to standardize style across docs

### CODE REVIEW

File: userguide/tutorials/aws-container.adoc [hunk 1]
```diff
@@ -24,29 +27,40 @@ image:https://github.com/killbill/killbill-docs/raw/v3/userguide/assets/aws/ecs-
 
 If all goes well, you should see a success screen:
```
Comment (reshmabidikar) [inline userguide/tutorials/aws-container.adoc:29]: Looks like the success screen image is missing.

File: userguide/tutorials/aws-container.adoc [hunk 2]
```diff
@@ -24,29 +27,40 @@ image:https://github.com/killbill/killbill-docs/raw/v3/userguide/assets/aws/ecs-
...
 
-image:https://github.com/killbill/killbill-docs/raw/v3/userguide/assets/aws/ecs-cluster-success.png[align=center]
+=== Create a Fargate Task Definition
```
Comment (reshmabidikar) [inline userguide/tutorials/aws-container.adoc:30]: Looks like the success screen image is missing.

File: userguide/tutorials/aws-container.adoc [hunk 3]
```diff
@@ -24,29 +27,40 @@ image:https://github.com/killbill/killbill-docs/raw/v3/userguide/assets/aws/ecs-
 
 If all goes well, you should see a success screen:
```
Comment (reshmabidikar) [inline userguide/tutorials/aws-container.adoc:28]: Looks like the success screen image (`https://github.com/killbill/killbill-docs/raw/v3/userguide/assets/aws/ecs-cluster-success.png`) is missing.

File: userguide/tutorials/aws-container.adoc [hunk 4]
```diff
@@ -24,29 +27,40 @@ image:https://github.com/killbill/killbill-docs/raw/v3/userguide/assets/aws/ecs-
...
 
-image:https://github.com/killbill/killbill-docs/raw/v3/userguide/assets/aws/ecs-cluster-success.png[align=center]
+=== Create a Fargate Task Definition
+
+. Create a Fargate Task Definition:
 
 Then, create a Fargate Task Definition:
```
Comment (reshmabidikar) [inline userguide/tutorials/aws-container.adoc:37]: Screenshot missing (`image:https://github.com/killbill/killbill-docs/raw/v3/userguide/assets/aws/ecs-task-definition-fargate.png`)

File: userguide/tutorials/aws-container.adoc [hunk 5]
```diff
@@ -24,29 +27,40 @@ image:https://github.com/killbill/killbill-docs/raw/v3/userguide/assets/aws/ecs-
...
 
-image:https://github.com/killbill/killbill-docs/raw/v3/userguide/assets/aws/ecs-task-definition-fargate.png[align=center]
+[start=2]
+. At the bottom of the screen, select *Configure via JSON*:
```
Comment (reshmabidikar) [inline userguide/tutorials/aws-container.adoc:40]: Screenshot missing `https://github.com/killbill/killbill-docs/raw/v3/userguide/assets/aws/ecs-task-definition-via-json.png`

File: userguide/tutorials/aws-container.adoc [hunk 6]
```diff
@@ -24,29 +27,40 @@ image:https://github.com/killbill/killbill-docs/raw/v3/userguide/assets/aws/ecs-
...
 
 Go to the bottom and select Configure via JSON:
 
-image:https://github.com/killbill/killbill-docs/raw/v3/userguide/assets/aws/ecs-task-definition-via-json.png[align=center]
+[start=3]
+. Enter the content of the following JSON file:
 
 Enter the content of the following JSON file: https://killbill-amp.s3.amazonaws.com/task.json
```
Comment (reshmabidikar) [inline userguide/tutorials/aws-container.adoc:47]: `Enter the content of the following JSON file` text seems to be repeated

File: userguide/tutorials/aws-container.adoc (hunk 3 above)
Comment (ghost) [reply userguide/tutorials/aws-container.adoc:28]: Added the image reference

**Review by ghost — COMMENTED** [review]
I fixed the missing image reference for the success screen

File: userguide/tutorials/aws-container.adoc (hunk 4 above)
Comment (reshmabidikar) [reply userguide/tutorials/aws-container.adoc:37]: This screenshot is still missing (`image:https://github.com/killbill/killbill-docs/raw/v3/userguide/assets/aws/ecs-task-definition-fargate.png`)

Comment (ghost) [reply userguide/tutorials/aws-container.adoc:37]: I have added it back in.

**Review by reshmabidikar — APPROVED** [review]

**Comment by pierre** [comment]
Thanks!

**Pull request merged**